tools/cli/src/opt/kanidm.rs

```diff
@@ -49,6 +49,13 @@ pub struct CommonOpt {
     /// Log format (still in very early development)
     #[clap(short, long = "output", env = "KANIDM_OUTPUT", default_value = "text")]
     output_mode: OutputMode,
+    /// Skip hostname verification
+    #[clap(
```

Comment:
Should this be a hidden flag?

Reply:
🤷🏻 it's a configuration option, I don't see a reason to hide it.